packages/tests/src.ts/test-providers.spec.ts

```diff
@@ -1268,20 +1274,36 @@ describe("Test Hedera Provider", function () {
     }).timeout(timeout * 4);
 
     it('should submit signed transaction', async function() {
+        const account = {
+            "operator": {
+            "accountId": "0.0.1340",
+                "publicKey": "302a300506032b65700321004aed2e9e0cb6cbcd12b58476a2c39875d27e2a856444173830cc1618d32ca2f0",
+                "privateKey": "302e020100300506032b65700422042072874996deabc69bde7287a496295295b8129551903a79b895a9fd5ed025ece8"
+            },
+            "network": {
+            "35.231.208.148:50211": "0.0.3",
+                "35.199.15.177:50211": "0.0.4",
+                "35.225.201.195:50211": "0.0.5",
+                "35.247.109.135:50211": "0.0.6"
+          }
+        };
```

Comment:
Lets move this out side of the `it` at the beginning of the file and define only the properties we actually use